src/skin/skinloader.cpp

```diff
@@ -72,7 +74,11 @@ QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schem
     if (preview.isNull()) {
         preview.load(":/images/skin_preview_placeholder.png");
     }
-    preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
+    QSize scaledSize(static_cast<int>(640 * devicePixelRatio),
+            static_cast<int>(360 * devicePixelRatio));
+    preview.setDevicePixelRatio(devicePixelRatio);
+    preview = preview.scaled(scaledSize, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Comment:
This can be shortened to 
```suggestion
    preview = preview.scaled(QSize(640, 360) * devicePixelRatio, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Reply:
I didn't find docu on how to scale a QSize with just a factor. did you try it?
if yes, feel free to push the shorter version.

Reply:
> I didn't find docu on how to scale a QSize with just a factor.

https://doc.qt.io/qt-5/qsize.html#operator-2a-1

Reply:
Ha! thanks.